src/data/carbon_vintage.cairo

```diff
@@ -40,7 +35,6 @@ impl CarbonVintageTypeInto of Into<CarbonVintageType, felt252> {
             CarbonVintageType::Projected => 0,
```

Comment:
Should we start from 1 and keep 0 for "unset" to differentiate Vintages that are not set yet.

Reply:
Great idea! 
i didn't create an `Unset` CarbonVintageType, just put start from 0 to 1, should we that type?